templates/web/base/around/on_map_list_items.html

```diff
@@ -1,7 +1,14 @@
-[% IF on_map.size %]
-    [% FOREACH problem IN on_map %]
+[% all_reports = on_map.merge(around_map) %]
```

Comment:
Appending the lists end-to-end like this means the end result isn't sorted in a particularly useful way (not a new bug in this PR, I know!)
Seeing as `on_map` and `around_map` don't seem to be used on their own any more, is it worth doing this merge in the controller, and sorting the reports appropriately there?